File: google-play-api-plugin/src/main/java/com/dipien/google/play/api/task/PublishBundleTask.java
package com.dipien.google.play.api.task;

import com.dipien.google.play.api.App;
import com.dipien.google.play.api.GooglePlayPublisher;

public class PublishBundleTask extends BaseTask {
	
	public PublishBundleTask() {
		setDescription("Publish Bundle to Google Play");
	}

	@Override
	protected void onExecute(App app) {
		GooglePlayPublisher.publishBundle(app);
	}

}

File: google-play-api-plugin/src/main/java/com/dipien/google/play/api/task/PublishBundleTask.kt
package com.dipien.google.play.api.task

import com.dipien.google.play.api.App
import com.dipien.google.play.api.PublishingService

open class PublishBundleTask : BaseTask() {
    override fun onExecute(app: App) {
        PublishingService().publishBundle(app)
    }

    init {
        description = "Publish Bundle to Google Play"
    }
}
